account_invoice_rounding/account.py

```diff
@@ -174,13 +177,28 @@ def _compute_amount(self):
                     self.amount_untaxed = (
                         swedish_rounding['amount_untaxed'])
 
-    @api.one
+    @api.multi
     def _get_rounding_invoice_line_id(self):
+        self.ensure_one()
         lines = self.env['account.invoice.line'].search(
             [('invoice_id', '=', self.id),
              ('is_rounding', '=', True)])
         self.global_round_line_id = lines
 
+    @api.multi
+    def onchange_partner_id(self, type, partner_id, date_invoice=False,
+                            payment_term=False, partner_bank_id=False,
+                            company_id=False):
+        result = super(AccountInvoice, self).onchange_partner_id(
+            type, partner_id, date_invoice=date_invoice,
+            payment_term=payment_term, partner_bank_id=partner_bank_id,
+            company_id=company_id)
+        partner = self.env['res.partner'].browse(partner_id or False)
```

Comment:
what's the reason for `partner_id or False`? Better to check `partner_id` outside.

Reply:
thanks, fixed